src/app/modules/login/pages/login-page/login-page.component.ts

```diff
@@ -22,14 +25,30 @@ export class LoginPageComponent implements OnInit {
     this.authService
       .login(user.email, user.password)
       .then((result) => {
-        console.log(result);
         this.loading$.next(false);
-        this.authService.setUserStorage(result.user);
-        this.router.navigate(['/todo/list']);
+        this.handleLoginSuccess(result.user);
+      })
+      .catch(() => {
```

Comment:
After converting a promise to Observable you can use rxjs operators instead of these chain functions of Promise, why should use Observable (Rxjs) ? do a search ;)